doc/overview.md

```diff
@@ -32,50 +32,58 @@ our [Discourse forum](https://forum.generic-mapping-tools.org/c/questions/pygmt-
 
 These are conference presentations about the development of PyGMT (previously "GMT/Python"):
 
+-   "Accessing and Integrating GMT with Python and the Scientific Python Ecosystem".
+    2024.
+	Yvonne Fröhlich, Dongdong Tian, Wei Ji Leong, Max Jones, and Michael Grund.
+	Presented at *AGU 2024*.
+	doi:[10.6084/m9.figshare.28049495](https://doi.org/10.6084/m9.figshare.28049495)
+
+    ![](https://s3-eu-west-1.amazonaws.com/pfigshare-u-previews/51279815/thumb.png){.align-center width="50%"}
+
 -   "Geospatial Analysis & Visualization with PyGMT".
     2022.
     Max Jones, Wei Ji Leong, and Leonardo Uieda.
     Presented at *SciPy 2022*.
     doi:[10.6084/m9.figshare.20483793](https://doi.org/10.6084/m9.figshare.20483793)
 
-    ![](_static/scipy2022-youtube-thumbnail.jpg){.align-center width="80%"}
+    ![](https://s3-eu-west-1.amazonaws.com/pfigshare-u-previews/36664998/thumb.png){.align-center width="50%"}
```

Comment:
For the images files 2022 and older, they have already been commited to the git history, so removing them won't actually reduce the download size when doing a `git clone`. So you can revert the changes here and in the lines below.